styles.css

```diff
@@ -1 +1,50 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+.container {
```

Comment:
Nog een media breakpoint toevoegen:
`@media only screen and (max-width: 700px)`